content/workshops/cascading-layouts.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Cascading Layouts
+sub: A workshop on resilient CSS layouts, June 24-26
+feature: workshop
+author: miriam
+date: 2024-05-15
+action:
+  text: Register Now »
+  url: 'https://ti.to/pland/css-layout'
+image:
+  src: talks/layout-workshop.png
+  position: center
+  alt: |
+    A pink box labeled
+    extrinsic size (imposed from outside),
+    with overflowing blue text
+    that has a dashed box,
+    and says intrinsic size (from the content)
+tags:
+  - CSS
+  - Cascade
+  - Custom Properties
+  - Container Queries
+  - Layout
+  - CSSWG
+included:
+  - title: Interactive Sessions
+    text: Virtual or in-person
+    icon: icons/talk
+  - title: Dedicated Q&A
+    text: To gain a thorough understanding
+    icon: icons/messaging
+  - title: Cutting Edge Techniques
+    text: From a W3C Invited Expert
+    icon: icons/rocket
+  - title: Certificate of Completion
+    text: Documenting your education
+    icon: icons/ribbon
+summary: |
+  CSS Layout techniques have come a long way
+  since the early days of Responsive Design,
+  but many websites are still using
+  decade-old approaches
+  that require more fragile code
+  and more ongoing maintenance.
+  **Take your sites to the next level
+  with this 3-day workshop
```

Comment:
I wonder if we should say 3-session instead of day?

Reply:
```suggestion
  with this 3-session workshop
```